dartagnan/src/main/java/com/dat3m/dartagnan/wmm/axiom/Irreflexive.java

```diff
@@ -17,24 +16,22 @@ public Irreflexive(Relation rel) {
         super(rel);
     }
 
-    @Override
-    public TupleSet getEncodeTupleSet(){
-        TupleSet set = new TupleSet();
-        rel.getMaxTupleSet().stream().filter(Tuple::isLoop).forEach(set::add);
-        return set;
-    }
+	@Override
+	public TupleSet getEncodeTupleSet(){
+		return new TupleSet();
```

Comment:
Yep. Irreflexivity, similar to Emptiness, is fully described by its disabled set. Only Acyclicity is not, because it cannot be expressed in terms of disabling single edges.